README.md

```diff
@@ -63,7 +63,7 @@ npm start
 
 ## ☁️ <a id="cloud"></a>Cloud platforms
 
-[![Deploy on Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/callsmusic/remix)
+[![Deploy on Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/ChankitSaini/remix)
```

Comment:
```suggestion
[![Deploy on Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/roj1512/remix)
```

Reply:
> 

You should have done this earlier! Anyways thanks!

Reply:
@roj1512 Will Heroku allow the forked repository to deployed from a banned repository? I suggest you to use a separate repo for it.

Reply:
No idea. But if you commit my suggestion, I'll merge this.

Reply:
> No idea. But if you commit my suggestion, I'll merge this.
Okay.

Reply:
> No idea. But if you commit my suggestion, I'll merge this.

Just a suggestion don't use forked repo, import it.